scripts/manage-db.py

```diff
@@ -183,11 +245,12 @@ def _strip_multiple_spaces(string):
     elif action == 'downgrade':
         db.downgrade(options.version)
     elif action == 'extract':
-        if len(args) < 2:
-            extract_active_data(options.db)
-        else:
-            location = args[1]
-            extract_active_data(options.db, location)
+        with db.begin() as trans:
```

Comment:
Good idea using a transaction, we'll probably prevent some edge cases (eg: releases existing one moment but not the next) this way.